packages/block-library/CHANGELOG.md

```diff
@@ -2,10 +2,6 @@
 
 ## Unreleased
 
-### Breaking Change
```

Comment:
I understand this package was not released yet, so we can simply remove this breaking change.